src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Collection/DictionaryDefaultConverter.cs

```diff
@@ -2,23 +2,21 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
-using System.Diagnostics;
-using System.Diagnostics.CodeAnalysis;
 using System.Text.Json.Serialization.Metadata;
 
 namespace System.Text.Json.Serialization.Converters
 {
     /// <summary>
     /// Default base class implementation of <cref>JsonDictionaryConverter{TCollection}</cref> .
     /// </summary>
-    internal abstract class DictionaryDefaultConverter<TDictionary, TKey, TValue>
-        : JsonDictionaryConverter<TDictionary, TKey, TValue>
+    internal abstract class DictionaryDefaultConverter<TDictionary, TKey, TValue, IntermediateType>
```

Comment:
See my earlier feedback. I think this could regress application sizes without much tangible impact.